<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>计算机图形学作业</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/icons/cube.ico</normaloff>:/icons/icons/cube.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="Content" name="widget" native="true"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="NavLabel" name="labelModels">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>72</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>72</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">margin: 5px 0px 5px;
border: none;
border-right: 5px solid #ccc</string>
        </property>
        <property name="text">
         <string>模
型
列
表</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="NavLabel" name="labelLights">
        <property name="minimumSize">
         <size>
          <width>32</width>
          <height>72</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>32</width>
          <height>72</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">
               margin: 5px 0px 5px;
               border: none;
               border-right: 5px solid #ccc
             </string>
        </property>
        <property name="text">
         <string>光
源
列
表</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="open"/>
    <addaction name="save"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
    <addaction name="frontView"/>
    <addaction name="backView"/>
    <addaction name="leftView"/>
    <addaction name="rightView"/>
    <addaction name="topView"/>
    <addaction name="bottomView"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="axis"/>
    <addaction name="plane"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSetting"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>新建</string>
   </property>
  </action>
  <action name="open">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="frontView">
   <property name="text">
    <string>前视图</string>
   </property>
  </action>
  <action name="backView">
   <property name="text">
    <string>后视图</string>
   </property>
  </action>
  <action name="leftView">
   <property name="text">
    <string>左视图</string>
   </property>
  </action>
  <action name="rightView">
   <property name="text">
    <string>右视图</string>
   </property>
  </action>
  <action name="topView">
   <property name="text">
    <string>顶视图</string>
   </property>
  </action>
  <action name="bottomView">
   <property name="text">
    <string>底视图</string>
   </property>
  </action>
  <action name="perspective">
   <property name="text">
    <string>透视投影</string>
   </property>
  </action>
  <action name="ortho">
   <property name="text">
    <string>正交投影</string>
   </property>
  </action>
  <action name="axis">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>坐标轴</string>
   </property>
  </action>
  <action name="plane">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>平面</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NavLabel</class>
   <extends>QLabel</extends>
   <header>controls.h</header>
  </customwidget>
  <customwidget>
   <class>Content</class>
   <extends>QWidget</extends>
   <header>content.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
